app/src/org/commcare/activities/PromptUpdateActivity.java

```diff
@@ -36,7 +36,9 @@
     @Override
     public void onCreateSessionSafe(Bundle savedInstanceState) {
         super.onCreateSessionSafe(savedInstanceState);
-        refreshUpdateToPromptObject();
+        if (updateToPrompt == null) {
```

Comment:
this null check seems unnecessary to me because `updateToPrompt` will always be null here

Reply:
Ah nevermind I understand-- in the instance where `isBackEnabled` gets called before `onCreate`, it won't be null and we'd be doing the refresh twice. Sorry for slowness on the uptake..